README.md

```diff
@@ -82,6 +82,18 @@ You now have a working setup from which you can build your own telemetry test en
 
 ***
 
+## Docker image
+```text
+# Clone the repo and from the repo root
+# Build fat JAR
+./gradlew clean build fatJar -x test
+# Prepare definition YAML/JSON files and store them in the ./definitions directory
```

Comment:
I still think that there should be an option to load the definitions at runtime vs build time, i.e.

`docker run -v ${PWD}/definitions:/definitions` image

Reply:
Fixed @svrnm .